wagtail/contrib/redirects/templates/wagtailredirects/index.html

```diff
@@ -1,10 +1,10 @@
-{% extends "wagtailadmin/base.html" %}
+{% extends "wagtailadmin/generic/base.html" %}
 {% load i18n %}
 {% load static wagtailadmin_tags %}
 
 {% block bodyclass %}redirects{% endblock %}
 
-{% block titletag %}{% trans "Redirects" %}{% endblock %}
+<!-- {% block titletag %}{% trans "Redirects" %}{% endblock %} -->
```

Comment:
If we don't need this - we should remove it completely, not just comment it out.